plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile

```diff
@@ -46,11 +46,120 @@ States are the central elements of a state machine. A state has to be placed ins
+
+| *Event trigger:* | *Prerequisites for the transition to fire:* |
+| @ev@ | Event _ev_ occured. |
+| @ev_1, ev_2@ | At least one the events _ev_1_ and _ev_2_ occured. |
+| @after 3s@ | Three seconds after activating the source state have passed. |
+| @after 3s, ev@ | Three seconds have passed or event _ev_ occurred, whatever happens first. |
+| @after 3s, after 2s@ | Three seconds have passed or two seconds have passed, whatever happens first. :-) |
+| @every 3s@ | Three seconds after activating the source state have passed. |
+| @always@ | Transition fires always. |
+| @default@ | Transition fires always. |
+| @else@ | Transition fires always. |
+| @oncycle@ | Transition fires always. |
+
+h4(#guard-condition). Guard condition
+
+A _guard_ applies an additional condition for a transition to fire. It is not sufficient for the specified event to occur, the guard condition must also be fulfilled. It is written in square brackets following the event trigger. It is also possible to specify a guard without any event trigger. In this case the transition is always taken, as long as the guard condition evaluates to _true_.
+
+The following examples assume the _x_ is an integer variable and _isOpen_ is an operation.
+
+| *Transition specification:* | *Prerequisites for the transition to fire:* |
+| @ev [x == 3]@ | Event _ev_ occured and _x_ holds the value 3. |
+| @ev_1, ev_2 [x == 3]@ | At least one the events _ev_1_ and _ev_2_ occured and _x_ holds the value 3. |
+| @after 3s, ev [x == 3 && isOpen(7)]@ | Three seconds have passed or event _ev_ occurred, whatever happens first. Additionally, _x_ holds the value 3 and _isOpen(7)_ returns _true_. |
+
+Please note that the guard condition applies to the whole event trigger, not just to the event specification left from it. That is, the guard condition must be fulfilled regardless of which of the events occured. You might think of the event specifications being enclosed in parentheses, like @(ev_1, ev_2) [x == 3]@, although it would be illegal to actually write down these parentheses.
+
+It is not possible to have event-specific guards as in the illegal @ev_1 [x == 1], ev_2 [x == 2]@. If you need something like this, use multiple transitions with @ev_1 [x == 1]@ on one of them and @ev_2 [x == 2]@ on the other.
+
+h4(#effect). Effect
+
+A transition specification may contain one or more _actions_ that are executed when the transition fires, like assigning a value to a variable, calling an operation, or raising an event. All actions taken together are called the transition's _effect_.
+
+In the transition specification, the effect is written right from event trigger and guard and separated by a @/@ character from them. "Figure &quot;Transition effect syntax&quot;":#fig_transition-effect-syntax shows how the effect's syntax looks like.
+
+p(#fig_transition-effect-syntax). 
+!(standard-image)images/docu_grammar_030_transition-effect.png(Transition effect syntax)!
```

Comment:
It is a technical detail that we distinguish between expression and event raising expression. In the figure it should be only one; and then the figure does not tell you much anymore; I think it can be omitted.